.travis.yml

```diff
@@ -21,7 +21,5 @@ before_script:
 
 script:
   - chef --version
-  - cookstyle --version
```

Comment:
We have these here for troubleshooting what command versions were being run. They're quite useful since we're using nightlies on DK

Reply:
@tas50 does the `chef --version` output not specify these? My experience is that it does, at least, for `cookstyle` and `foodcritic`… huh. How about that; I’m running nightly now, and it doesn’t.

Returning them!

Reply:
All set; sorry about that. Coulda sworn it output those versions…

Reply:
❰tsmith❙~❱✔≻ /opt/chefdk/embedded/bin/chef --version                                                      13:11:13
Chef Development Kit Version: 3.0.29
chef-client version: 14.1.1
delivery version: master (7206afaf4cf29a17d2144bb39c55b7212cfafcc7)
berks version: 7.0.2
kitchen version: 1.21.2
inspec version: 2.1.68

Reply:
It wouldn't be a bad idea to add them there, but atm they are not there